mattermost4j-core/src/main/java/net/bis5/mattermost/client4/MattermostClient.java

```diff
@@ -247,15 +257,17 @@ public MattermostClient(String url) {
    * Create new MattermosClient instance.
    */
   public MattermostClient(String url, Level logLevel) {
-    this(url, logLevel, false);
+    this(url, logLevel, false, clientBuilder -> {});
   }
 
-  MattermostClient(String url, Level logLevel, boolean ignoreUnknownProperties) {
+  MattermostClient(String url, Level logLevel, boolean ignoreUnknownProperties,
+    Consumer<ClientBuilder> httpClientConfig)
+  {

```

Comment:
We currently use [Google's Java coding style](https://google.github.io/styleguide/javaguide.html).
If you uses Eclipse or IntelliJ, get formatter configuration from [here](https://github.com/google/styleguide/) and apply to these codes.
Otherwise, please use K&R style braces (no line break before the opening brace).